tests_e2e/tests/ext_sequencing/ext_sequencing.py

```diff
@@ -226,7 +226,7 @@ def run(self):
                 # fail. We know an extension should fail if "failing" is in the case name. Otherwise, report the
                 # failure.
                 deployment_failure_pattern = r"[\s\S]*\"details\": [\s\S]* \"code\": \"(?P<code>.*)\"[\s\S]* \"message\": \"(?P<msg>.*)\"[\s\S]*"
-                msg_pattern = r"Multiple VM extensions failed to be provisioned on the VM. Please see the VM extension instance view for other failures. The first extension failed due to the error: VM Extension '.*' is marked as failed since it depends upon the VM Extension 'CustomScript' which has failed."
+                msg_pattern = r"Multiple VM extensions failed to be provisioned on the VM.*VM Extension '.*' is marked as failed since it depends upon the VM Extension 'CustomScript' which has failed."
```

Comment:
should we also match "VM Extension 'AzureMonitorLinuxAgent' is marked as failed "?

Reply:
The next test case expects RunCommandLinux to fail because it's dependent on Custom script. The pattern matches for both test cases

Reply:
maybe we should parameterize the extension name?

Reply:
the test would be more robust if we check the extension that failed and the extension we did not execute

Reply:
We could, but I'm not sure it matters. This check mainly exists to confirm that the deployment failure is due to extension dependency failures, and not some other failure. 
There's additional checks below this one that checks the agent log to confirm the correct extensions failed due to their dependencies.